x-pack/agent/pkg/agent/program/testdata/single_config-metricbeat.yml

```diff
@@ -1,11 +1,15 @@
 metricbeat:
   modules:
   - module: docker
-    setting: one
+    metricsets: [status]
+    index: metricbeat-docker.status-default
   - module: apache
-    setting: two
+    metricsets: [info]
+    index: metricbeat-generic-testing
```

Comment:
should be metrics-{dataset}-{namespace}

Reply:
namespace==testing in this case so i think it is correct.